SOURCE/FD_Jacobian.c

```diff
@@ -67,38 +67,27 @@ void PhaseRheologyLoop( int centroid, double sign, double denom, double Exx, dou
         }
         
         if ( model->eta_avg == 0) { // ARITHMETIC AVERAGE
-            if ( cond == 1 ) detadE[c0]      += sign*vol[p][c0] * etaVE/(denom);
+            if ( cond == 1 ) detadE[c0]      += sign*vol[p][c0] * etaVE/(denom); // TODO: reformat etaVE to eta_eff
```

Comment:
What does cond mean?
Reading `fabs(vol[p][c0])>1.0e-13` does not making this boolean clearer

Reply:
1.0e-13 should be a constant with the name

Reply:
cond should be also constant but initialised inside the cycle

Reply:
I have reformatted this and also applied the changes in the function `NonNewtonianViscosityGrid()` (RheologyDensity.c)